Bukkit/src/main/java/com/plotsquared/bukkit/placeholder/Placeholders.java

```diff
@@ -181,6 +187,31 @@ public Placeholders() {
             default:
                 break;
         }
+        final Matcher matcher = this.flagPattern.matcher(identifier);
+        if (matcher.find()) {
+            return getFlag(plot, matcher.group(1));
+        }
+        return "";
+    }
+
+    /**
+     * Return the flag value from its name on the current plot.
+     * If the flag doesn't exist it return an empty string.
+     * If the flag exists but it is not set on current plot it return the default value.
+     *
+     * @param plot     Current plot where the player is
+     * @param flagName Name of flag to get from current plot
+     * @return The value of flag serialized in string
+     */
+    private String getFlag(final Plot plot, final String flagName) {
+        final PlotFlag<?, ?> flag = GlobalFlagContainer.getInstance().getFlagFromString(flagName);
+        if (flag != null) {
+            return plot.getFlags().stream()
```

Comment:
That should give the same result as `plot.getFlag(flag).toString()`

Reply:
Change applied in last commit